core/libp2p/connection/stream.hpp

```diff
@@ -6,19 +6,62 @@
 #ifndef KAGOME_CONNECTION_STREAM_HPP
 #define KAGOME_CONNECTION_STREAM_HPP
 
-#include "libp2p/basic/readwritecloser.hpp"
+#include <continuable/continuable.hpp>
```

Comment:
I should have told you about continuable config, sorry.
Please use `libp2p/basic/continuable.hpp` instead

It enables use of error code instead of exceptions